KubeArmor/core/kubeArmor.go

```diff
@@ -446,12 +451,44 @@ func KubeArmor() {
 		}
 	}
 
-	// == //
+	trimmedSocket := trimmedSocketPath()
```

Comment:
I am not sure why we are calling `trimmedSocketPath()` at this point?
`trimmedSocket` seems to be used only `if !dm.K8sEnabled && cfg.GlobalCfg.Policy`.

Reply:
Actually we are also using it here: https://github.com/Ankurk99/KubeArmor/blob/main/KubeArmor/core/kubeArmor.go#L492